project/Build.scala

```diff
@@ -746,6 +759,21 @@ object Build {
         (unmanagedSourceDirectories in (`dotty-library-bootstrapped`, Compile)).value,
     )
 
+  lazy val `dotty-staging` = project.in(file("staging")).
+    withCommonSettings(Bootstrapped).
+    dependsOn(dottyCompiler(Bootstrapped)).
+    dependsOn(dottyCompiler(Bootstrapped) % "test->test").
+    settings(commonBootstrappedSettings).
+    settings(commonDottyJarClasspathSettings).
+    settings(
+      fork in run := true,
```

Comment:
This line and the next should no longer be needed, this is now the default.

Reply:
removed